lib/tasks/gobierto_budgets/fixtures.rake

```diff
@@ -114,6 +114,8 @@ namespace :gobierto_budgets do
     def import_categories
       puts "== Importing categories =="
       categories = categories_fixtures do |category|
+        category.merge!('kind' => category['kind'] == 'I' ? 'income' : 'expense')
```

Comment:
Regarding this same change, I'd mention that a complete `script/setup` is required in order to keep things working. Not big deal though.